Graphs/TreeGraph/index.js

```diff
@@ -580,13 +577,15 @@ class TreeGraph extends AbstractGraph {
         } = this.getConfiguredProperties();
 
         const {
-            renderNode
+            renderNode,
+            currentUser,
+            currentEnterprise,
         } = this.props;
 
         if (!isFunction(renderNode)) return '<div/>';
 
         const rectColorText = d.data.clicked ? rectNode.selectedTextColor : rectNode.defaultTextColor
-        return renderNode({data: d.data, textColor: rectColorText, nodeId: `node-${d.id}`, ...rectNode});
+        return renderNode({data: d.data, textColor: rectColorText, nodeId: `node-${d.id}`, currentUser, currentEnterprise, ...rectNode});
```

Comment:
@myTechPlayArea This is a lib we cannot pass to it currentUser/currentEnterprise! You should have a customProps that can be injected into the node to render!